src/frontends/paddle/src/op/flip.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs flip(const NodeContext& node) {
+    const auto data_node = node.get_input("X");
+    const auto axes = node.get_attribute<std::vector<int32_t>>("axis");
+    const auto input_shape = data_node.get_partial_shape().get_shape();
+    const auto dims = static_cast<int32_t>(data_node.get_partial_shape().rank().get_length());
+    const auto dtype = data_node.get_element_type();
+    // for zero-dim input
+    PADDLE_OP_CHECK(node, (dims > 0), "Input dims must be greater than 0");
+
+    Output<Node> temp = data_node;
+    std::vector<Output<Node>> temp_split_out;
+    int32_t axis;
+    for (size_t idx = 0; idx < axes.size(); idx++) {
```

Comment:
is it possible to use Reverse or ReverseSequence openvino operators?

Reply:
Exactly, so we do not need to manipulate with input shape and support this operation for dynamic shape input as well

Reply:
> Exactly, in this case we do not need to manipulate with input shape and we will be able to support this operation for dynamic shape input as well

Hi, I find that `flip` is the same as [reverse](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/paddle/src/op/reverse.cpp).
So do I still need to add flip?

Reply:
Sure, just to re-implement the translator using `ReverseSequence`. `Reverse` looks deprecated some time ago: https://docs.openvino.ai/latest/openvino_docs_ops_opset10.html#doxid-openvino-docs-ops-opset10

Reply:
Hi, I have update the codes, but I found it's still hard to support dynamic shape input.
Because `ReverseSequence` need the  length to reverse in specified seq_axis.
This requires input shape to being static.